### PR TITLE
Added Peter Kovesi's awesome colormap test image

### LINKED ISSUE #10: Add Peter Kovesi's awesome test image
http://arxiv.org/abs/1509.03700

### PR DESCRIPTION
**Pull request by jerryz123**
Peter Kovesi's test image is now a sample image that is displayed.

closes #10

### CODE REVIEW

**Comment by njsmith** [comment]
Hi @jerryz123! Thanks for working on this.

There are two main issues I see with the initial version of this patch:
- There are lots code formatting issues -- stuff like putting consistent spaces around operators. In Python we generally follow the rules described in [PEP 8](https://www.python.org/dev/peps/pep-0008/) (and there are tools like flake8 that will check this automatically -- depending on what editor you use it's very likely that there's some way you can set it up to automatically highlight incorrect formatting as you type). You might think, this is trivial, what does it matter? But this is the wrong way to think about it. Instead you should think, this is trivial, so I should get it right :-). There are lots of things in programming that are difficult and complex and out-of-our-control, so it's important not to trip ourselves up with things that we _can_ control. Also, code readability is incredibly important -- I would say more important than speed or correctness, even, because if code is readable you can make it fast or find bugs, but if it's not readable then soon it will be slow and buggy because no-one can maintain it :-). So it's worth training yourself to just always use correct code formatting -- pretty soon it becomes a habit and you don't even have to think about it.
- The way you're storing the new test image (with both the `.asc` and the `.txt` file seems to have added ~a megabyte to the source tree. This is a bit much :-). Looking at Peter's download page, there's also a small `.tif` file which can be read with `import matplotlib.pyplot as plt; data = plt.imread("colourmaptest.tif")`. I'm not 100% sure that matplotlib is guaranteed to be able to read `.tif` files though (I think it might depend on having other optional packages installed as well, like `Pillow`), so if we want to be sure that this works everywhere we could switch to another terse format, like numpy's native `npy` format: `np.save("colourmaptest.npy", plt.imread("colourmaptest.tf"))`. This produces a nice little 65 KiB file that can be read with `np.load`.

(BTW, since I was writing this, I decided to run flake8 over the existing source tree, and it did flag a few problems :-). Most of them were places where the tool was just [being a bit too simpleminded](https://www.python.org/dev/peps/pep-0008/#a-foolish-consistency-is-the-hobgoblin-of-little-minds), but I did fix up some that were more legitimate, and it looks like in the process I created a conflict with your changes in this branch. If you're comfortable with git you could use rebase or merge to fix the conflicts, or given the second issue it might be easier to just make a new branch off of master and re-implement the changes there.)

File: viscm/examples/ascconvert.py [hunk 1]
```diff
@@ -0,0 +1,20 @@
+# Simple program to convert ASCII grid format test image to 2D matrix text file to be loaded by numpy
+
+def concat(l):
+    s=""
+    for i in l:
+        s=s+i+" "
+    return s
```
Comment (njsmith) [inline viscm/examples/ascconvert.py]: BTW, check out Python's `str.join` method :-)

**Comment by tacaswell** [comment]
mpl delegates tiffs to `Pillow`.

Shouldn't it be possible to compute the test image on demand?

**Comment by jerryz123** [comment]
refer to #12

**Pull request closed without merging**